plugins/woocommerce/tests/metrics/config/global-setup.ts

```diff
@@ -0,0 +1,23 @@
+import { request } from '@playwright/test';
```

Comment:
Most of these setup files are copied and adapted from the Gutenberg repository. There's probably an opportunity to share some of this through `@wordpress/scripts` but it seems out of scope.